<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Danta</class>
 <widget class="QMainWindow" name="Danta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>단타쟁이 v0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>danta.png</normaloff>danta.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>281</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label_account">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>35</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_quantity">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>203</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_price">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>235</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_kinds">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_order">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_event">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>94</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_account">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_order">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_kinds">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>165</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_event">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_event_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_quantity">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>198</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_price">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
